app/src/main/java/com/internship/move/feature/authentication/UserResponse.kt

```diff
@@ -0,0 +1,15 @@
+package com.internship.move.feature.authentication
+
+import android.os.Parcelable
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kotlinx.parcelize.Parcelize
+
+@JsonClass(generateAdapter = true)
+@Parcelize
+data class UserResponse(
```

Comment:
I've seen that you use the `UserResponse` in many places. This is not right.

The `UserResponse` should have only 1 responsibility: to receive the data from the backend. It shouldn't reach the UI layer